engine/client/s_main.c

```diff
@@ -1860,6 +1863,33 @@ void S_SoundInfo_f( void )
 	S_PrintBackgroundTrackState ();
 }
 
+/*
+=================
+S_VoiceRecordStart_f
+=================
+*/
+void S_VoiceRecordStart_f( void )
+{
+	if( cls.state != ca_active || cls.legacymode )
```

Comment:
Since there is a check for legacy mode servers, we absolutely don't need any `svc_voicedata/init` parsing for legacy protocol above.